tests/main.cpp

```diff
@@ -54,23 +54,17 @@ TEST_CASE("Simulate & Send Transaction") {
   CHECK_EQ(transactionSignature, b58Sig);
 }
 
-TEST_CASE("Request Airdrop") {
+TEST_CASE("Request Airdrop & Confirm Transaction") {
   const solana::Keypair keyPair = solana::Keypair::fromFile(KEY_PAIR_FILE);
   const auto connection = solana::rpc::Connection(solana::DEVNET);
   // request Airdrop
   const auto prev_sol = connection.getBalance(keyPair.publicKey);
   const auto signature = connection.requestAirdrop(keyPair.publicKey, 50001);
   uint8_t timeout = 15;
-  // check signature status
-  // this is a temporary fix. This will be changed to the confirmTransaction
-  // function call once it gets implemented
-  while (timeout > 0) {
-    const auto res = connection.getSignatureStatus(signature, true).value;
-    if (res.has_value() && res.value().confirmationStatus == "finalized") {
-      break;
-    }
-    timeout--;
-    std::this_thread::sleep_for(std::chrono::seconds(1));
+  // using confirmTransaction to check if the airdrop went through
+  bool confirmation = false;
+  while (!confirmation) {
```

Comment:
Inifinte loop, avoid doing infinite loops even for tests.

Reply:
fixed